src/TurbulenceConvection.jl

```diff
@@ -69,7 +69,7 @@ include("NetCDFIO.jl")
 include("ReferenceState.jl")
 include("types.jl")
 
-include("thermodynamic_functions.jl")
+include("anelastic_thermodynamic.jl")
```

Comment:
```suggestion
include("anelastic_thermodynamics.jl")
```
And change filename accordingly